src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  margin: 0 auto;
+  margin-top: 70px;
+  padding: $calendar-padding;
+  gap: $cell-gap;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $cell-width;
+    height: $cell-height;
+
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: $number-size;
+    border: 1px solid black;
+    background-color: $cell-color;
+    transition: transform 0.5s, background-color 0.5s;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i}) {
+
+        &::before {
```

Comment:
Better write this selector in 1 line, kind of this: `&:nth-child(#{$i})::before`